src/Me/components/Button/Button.test.js

```diff
@@ -1,25 +1,39 @@
+import { render } from '@testing-library/react';
```

Comment:
If you already added `onClick`, maybe you can add a test for clicking on the button.

Reply:
Wouldn't that be considered an implementation detail?

Reply:
Nope. The component declare it accepts `onClick` callback and fire it when the component clicked. This should be tested.

Reply:
ok